src/ring_theory/ideal/operations.lean

```diff
@@ -1670,9 +1592,8 @@ assume a b, quotient.induction_on₂' a b $
   assume a b (h : f a = f b), ideal.quotient.eq.2 $
 show a - b ∈ ker f, by rw [mem_ker, map_sub, h, sub_self]
 
-lemma lift_injective_of_ker_le_ideal (I : ideal R) {f : R →+* S}
-  (H : ∀ (a : R), a ∈ I → f a = 0) (hI : f.ker ≤ I) :
-  function.injective (ideal.quotient.lift I f H) :=
+lemma lift_injective_of_ker_le_ideal (I : ideal R) {f : R →+* S} (H : ∀ (a : R), a ∈ I → f a = 0)
+  (hI : f.ker ≤ I) : function.injective (ideal.quotient.lift I f H) :=
```

Comment:
This is another example where I feel the original formatting got it exactly right: one line for theorem statement and type of the dependent variables, then a line with the hypotheses, then a line with the conclusions. 

In particular, I much prefer breaking lines at the `:` between hypothesis and conclusion, instead of in the middle of hypotheses.

```suggestion
lemma lift_injective_of_ker_le_ideal (I : ideal R) {f : R →+* S}
  (H : ∀ (a : R), a ∈ I → f a = 0) (hI : f.ker ≤ I) :
  function.injective (ideal.quotient.lift I f H) :=
```